compiler/luci-interpreter/src/kernels/StridedSlice.cpp

```diff
@@ -44,7 +44,7 @@ void StridedSlice::configure()
   assert(begin()->element_type() == DataType::S32);
   assert(end()->element_type() == DataType::S32);
   assert(strides()->element_type() == DataType::S32);
-  assert(input()->shape().num_dims() <= 4);
+  assert(input()->shape().num_dims() <= 5);
```

Comment:
It is already supported in TF2.8 - https://github.com/tensorflow/tensorflow/blob/3f878cff5b698b82eea85db2b60d65a2e320850e/tensorflow/lite/kernels/internal/strided_slice_logic.h#L39
(introduced in https://github.com/tensorflow/tensorflow/commit/bd9e81f2a8be1de26c4e9a8a3180921afa8e5dd7)